test/src/kbasesearchengine/test/main/IndexerWorkerTest.java

```diff
@@ -1050,13 +1051,273 @@ public SourceData answer(final InvocationOnMock inv) throws Throwable {
                         "GUID code:4/5/6 not found"))));
         
         verify(idxStore, never()).indexObjects(
-                any(), any(), any(), any(), any(), any(), anyBoolean());
+                any(), any(), any(), any(), any(), any());
         
         verify(storage).store(eq(event), eq("GUID_NOT_FOUND"), argThat(new ThrowableMatcher(
                 new UnprocessableEventIndexingException(ErrorType.GUID_NOT_FOUND,
                         "GUID code:4/5/6 not found"))));
     }
-    
+
+    @Test
+    public void indexDependentGUIDPublic() throws Exception {
+        /* tests the checking if the dependent object is indexed with the right permission. */
```

Comment:
typo
 /* tests if the dependent object is indexed with the right permission. */

Reply:
Done